cmake/manifoldDeps.cmake

```diff
@@ -33,6 +33,10 @@ function(logmissingdep PKG)
   endif()
 endfunction()
 
+set(OLD_BUILD_SHARED_LIBS ${BUILD_SHARED_LIBS})
+# we build fetched dependencies as static library
+set(BUILD_SHARED_LIBS OFF)
```

Comment:
Note: When building as static libs, the libraries become bundled into the dynamic libmanifold library. This means that any attempt to locate the static library will fail, so things like `find_package(Clipper2 REQUIRED)` in `manifoldConfig.cmake` would also need to be removed for this build variant

Reply:
..and we need to remove those libraries from the public link interface of the manifold target (PUBLIC -> PRIVATE)